slack-api-client/src/test/java/test_with_remote_apis/ProxyTest.java

```diff
@@ -11,41 +11,129 @@
 import com.slack.api.util.http.SlackHttpClient;
 import config.Constants;
 import lombok.extern.slf4j.Slf4j;
-import org.junit.Ignore;
+import org.eclipse.jetty.proxy.ConnectHandler;
+import org.eclipse.jetty.proxy.ProxyServlet;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
+import util.PortProvider;
 
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.hamcrest.CoreMatchers.*;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
 
 @Slf4j
 public class ProxyTest {
 
-    static String proxyUrl = "http://localhost:8888";
-
     static SlackConfig config = new SlackConfig();
-
-    static {
-        config.setProxyUrl(proxyUrl);
-    }
-
-    static Slack slack = Slack.getInstance(config);
+    static Server server = new Server();
+    static ServerConnector connector = new ServerConnector(server);
 
     static String botToken = System.getenv(Constants.SLACK_SDK_TEST_BOT_TOKEN);
     static String rtmBotToken = System.getenv(Constants.SLACK_SDK_TEST_CLASSIC_APP_BOT_TOKEN);
     static String scimToken = System.getenv(Constants.SLACK_SDK_TEST_GRID_ORG_ADMIN_USER_TOKEN);
     static String auditLogsToken = System.getenv(Constants.SLACK_SDK_TEST_GRID_ORG_ADMIN_USER_TOKEN);
 
-    @Ignore
```

Comment:
When I wrote this test, I manually ran the code and verified it fails. I've improved all the tests in this class to correctly verify if the clients work with the enabled local proxy server.